docs/setup_eecs281.md

```diff
@@ -80,41 +80,52 @@ If you've used Visual Studio on your computer before, start the tutorial at the
 
 If Visual Studio is new to you or new to your computer, start at the [beginning](https://eecs280staff.github.io/p1-stats/setup_visualstudio.html).  Stop after you've completed the [Add existing files](https://eecs280staff.github.io/p1-stats/setup_visualstudio.html#add-existing-files) section.
 
-#### Getopt Setup
+#### `getopt` library
+Visual Studio does not ship with a `getopt` library, which processes command line arguments.  We'll install a copy.
 
-Next, we will make sure to add the `getopt` library to Visual Studio. `getopt` is a C/C++ library used to process command line arguments, and its the library that we will be using in EECS 281. While the other IDEs contain of version of `getopt` in their compiler, Visual Studio does not.
+First, download [`getopt.c`](getopt.c) and [`getopt.h`](getopt.h).
 
-First, please download the `getopt.c` file <a href="/getopt.c" download>here</a> and the `getopt.h` file  <a href="/getopt.h" download>here</a>.
+##### Option 1 (Recommended)
```

Comment:
This is much nicer than having a list of the two options! Thanks for the change!